Directory.Packages.props

```diff
@@ -18,7 +18,7 @@
   </ItemGroup>
   <ItemGroup>
     <GlobalPackageReference Include="GitHubActionsTestLogger" Version="2.3.3" />
-    <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
-    <GlobalPackageReference Include="PolySharp" Version="1.13.2" />
+    <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
```

Comment:
According to [the docs](https://github.com/dotnet/sourcelink#:~:text=If%20your%20project%20uses%20.NET%20SDK%208%2B%20and%20is%20hosted%20by%20the%20above%20providers%20it%20does%20not%20need%20to%20reference%20any%20Source%20Link%20packages%20or%20set%20any%20build%20properties.) sourcelink reference is no longer needed if it is using .NET 8 SDK
```suggestion
```